src/js/components/DataFilter/__tests__/DataFilter-test.tsx

```diff
@@ -56,6 +56,21 @@ describe('DataFilter', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('renders outside grommet wrapper', () => {
+    const { container } = render(
+        <Data data={data}>
+          <DataFilters>
+            <DataFilter property="name" />
+            <DataFilter property="enabled" />
+            <DataFilter property="rating" />
+            <DataFilter property="blank" />
+            <DataFilter property="zero" />
+          </DataFilters>
+        </Data>
+    );
+    expect(container.firstChild).toMatchSnapshot();
+  });
```

Comment:
Should we remove this test since DataFilter is wrapped by Data and DataFilters and will get the theme from those, so we can't really test only the DataFilter?